kmir/src/kmir/kmir.py

```diff
@@ -73,28 +75,86 @@ def make_init_config(
         init_config = subst.apply(self.definition.init_config(KSort(sort)))
         return init_config
 
+    def make_call_config(
+        self, parsed_smir: KApply, smir_info: SMIRInfo, start_symbol: str = 'main', sort: str = 'GeneratedTopCell'
+    ) -> tuple[KInner, list[KInner]]:
+
+        if not start_symbol in smir_info.function_tys:
+            raise KeyError(f'{start_symbol} not found in program')
+
+        _sym, _allocs, functions, items, types, _ = parsed_smir.args
+
+        args_info = smir_info.function_arguments[start_symbol]
+
+        locals, constraints = symbolic_locals(smir_info, args_info)
+
+        subst = {
+            'K_CELL': mk_call_terminator(smir_info.function_tys[start_symbol], len(args_info)),
+            'STARTSYMBOL_CELL': KApply('symbol(_)_LIB_Symbol_String', (stringToken(start_symbol),)),
+            'STACK_CELL': list_empty(),  # FIXME see #560, problems matching a symbolic stack
+            'LOCALS_CELL': list_of(locals),
+            'FUNCTIONS_CELL': KApply(
+                'mkFunctionMap',
+                (
+                    functions,
+                    items,
+                ),
+            ),
+            'TYPES_CELL': KApply(
+                'mkTypeMap',
+                (
+                    map_empty(),
+                    types,
+                ),
+            ),
+            'ADTTOTY_CELL': KApply(
+                'mkAdtMap',
+                (
+                    map_empty(),
+                    types,
+                ),
+            ),
```

Comment:
Each of these can probably be a single line, no need to amke it take up so much vertical space.

I would make them a single line each, and then use `make format` to decide which needs to be broken. The formatter should make the decisions about line lengths.

Reply:
I agree - the formatter did this vertical layout, actually...

Reply:
Does collapsing the tuple `(map_empty(), types,)` work?

Reply:
No, didnt. But as you are mentioning this I noticed the comma behind `types` there.
Removed that comma and the formatter accepted the single-line version :tada:  ).